docs/guides/integration-prebid.md

```diff
@@ -1,172 +1,230 @@
-- [Integration Steps](#integration-steps)
-- [Generate UID2 Token](#generate-uid2-token)
-- [UID2 User ID Submodule](#uid2-user-id-submodule)
-- [Client Refresh Mode](#client-refresh-mode)
-  -  [Response Storage Options](#response-storage-options)
-  -  [Client Refresh Cookie Example](#client-refresh-cookie-example)
-  -  [Client Refresh uid2Token Example](#client-refresh-uid2token-example)
-- [Storage of Internal Values](#storage-of-internal-values)
-- [Sample Token](#sample-token)
-- [Prebid Implementation Notes and Tips](#prebid-implementation-notes-and-tips)
-- [Configuration Parameters for `usersync`](#usersync-configuration-parameters) -->
+This guide does not apply to publishers that want to use a [private operator](../ref-info/glossary-uid.md#gl-private-operator), or those that want to generate tokens server-side.
+Those publishers should follow the [Prebid.js Advanced Integration Guide](./integration-prebid-advanced.md).
 
-This guide is for publishers who want to integrate with UID2 and generate [UID2 tokens](../ref-info/glossary-uid.md#gl-uid2-token) (advertising tokens) to be passed by Prebid in the RTB bid stream.
+UID2 provides a [Prebid.js module](https://docs.prebid.org/dev-docs/modules/userid-submodules/unified2.html) with the following features:
 
-It outlines the basic steps to consider if you're building a direct integration with UID2 and you use Prebid for header bidding. 
+- UID2 token generation.
+- Automatic refreshing of UID2 tokens.
+- Automatic storage of UID2 tokens in the browser.
+- Automatic passing of UID2 tokens to the bid stream.
 
-## Introduction
+In order to follow this guide you will need to make changes to the HTML and JavaScript on your site. No server-side work is required.
 
-If you're a publisher using Prebid for header bidding, there are a few extra steps to take so that your Prebid header bidding implementation also supports UID2.
+To integrate with UID2 using Prebid.js you will need to complete the following steps:
 
-In addition, if you don't already have one, you must set up a UID2 account: see [Account Setup](../getting-started/gs-account-setup.md).
+1. Complete the [UID2 Account Setup](../getting-started/gs-account-setup.md).
+2. Add Prebid.js to your site.
+3. Configure the UID2 module.
 
-## UID2 Prebid Module Page
+## Complete UID2 Account Setup
 
-Information about how to integrate Prebid with UID2 is also in these locations:
-- On the [Unified ID 2.0](https://docs.prebid.org/dev-docs/modules/userid-submodules/unified2.html) page for the Prebid User ID submodule on the Prebid site.
-- In the [UID2 User ID Submodule](https://github.com/prebid/Prebid.js/blob/master/modules/uid2IdSystem.md) page in the Prebid GitHub repository.
+Complete the UID2 account setup by following the steps described in the [Account Setup](../getting-started/gs-account-setup.md) page. You will need to provide a list of domain names for the sites that you will be using with Prebid.js.
```

Comment:
Can we add "You will need to provide a list of domain names for the sites that you will be using with Prebid.js." to after when the account setup is complete? Feels like the domain names is part of the actual account setup. 

Complete the UID2 account setup by following the steps described in the [Account Setup] page. 
When account setup is complete you will be provided with a public key and subscription ID. These will be used to configure the UID2 module. You will also need to provide a list of domain names for the sites that you will be using with Prebid.js.

Reply:
Pubs will have to provide domain names as part of the account setup, otherwise the public key and subscription ID they receive won't be usable

Reply:
Added some clarification

> You will need to provide a list of domain names for the sites that you will be using with Prebid.js as part of the account setup process.

Reply:
This sounds better, thank you!